### ISSUE TITLE
Why the batch normalization layer is omitted in your implementation

### ISSUE THREAD

**Issue by shanyucha** [toimcio/SegNet-tensorflow#8]
In the original implementation of caffe-segnet, the bn layer is appended after conv layer. So why this implementation did not use batch normalization layer?

**Comment by Bonnie970** [comment]
https://github.com/toimcio/SegNet-tensorflow/blob/4f925b3ab3d66adf2e378c2655c115e4d7ccc65b/layers_object.py#L59 

There is BN wrapped in cnn layer.
